packages/node-sdk/src/utils.ts

```diff
@@ -14,6 +14,13 @@ export function ok(condition: boolean, message: string): asserts condition {
   }
 }
 
+export function assertId(id: string | number, entity: string) {
```

Comment:
Nit: To keep consistent maybe name it idOk

Reply:
renamed!